ext/scheduler/airflow/dag/compiler_test.go

```diff
@@ -129,6 +129,7 @@ func setupJobDetails(tnnt tenant.Tenant) *scheduler.JobWithDetails {
 		EndDate:       &end,
 		Interval:      "0 2 * * 0",
 		DependsOnPast: true,
+		CatchUp:       false,
```

Comment:
let's set up for catch up with value `true` as well. this way we can have a confirmation in the expected compiled dag. so that if there's any error during compilation, it will be catched ehre.